ansible/continue.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Execute getbestchainlock
+  hosts: wallet_nodes
+  tasks:
+    - name: Run dash-cli getbestchainlock
+      ansible.builtin.command: dash-cli getbestchainlock
+      register: chainlock_output
+      ignore_errors: true
+      no_log: true
+
+    - name: Check for chainlock
+      ansible.builtin.fail:
```

Comment:
`fail` shouldn't be necessary if we just remove the `ignore_errors` from previous step and/or add `failed_when`. But even that shouldn't be necessary, it will fail anyway when `chainlock_output.rc != 0` right?

Reply:
In future, we will need to check exactly why chainlocks aren't there. IE: are the sporks on? If no... turn them on then fail, if yes, just fail.

This is just a template for that behavior from v20